src/app/members/doctor/profile/profile.page.html

```diff
@@ -112,7 +112,14 @@
       <ion-label color="primary" position="floating">PMDC</ion-label>
       <ion-input type="text" formControlName="PMDC" required></ion-input>
     </ion-item>
-
+    <ion-item>
+      <ion-label color="primary" position="floating">Additional Info</ion-label>
+      <ion-input type="text" formControlName="AdditionalInfo" required></ion-input>
```

Comment:
@themuhammadattaurrehman 
Talk to moiz, he added some validation in singup and add those validation here too